metaflow/plugins/kfp/kfp_s3_sensor.py

```diff
@@ -7,28 +7,79 @@
 from email.policy import default
 import click
 
+import os
 import pathlib
 from typing import Dict
+import botocore
+from botocore.exceptions import ClientError
+import base64
+import json
+import marshal
+import time
+from urllib.parse import urlparse, ParseResult
+
+from typing import Tuple
+
+from metaflow.datastore.util.s3util import get_s3_client
+
+
+def construct_elapsed_time_s3_bucket_and_key(
+    flow_name: str, kfp_run_id: str
+) -> Tuple[str, str]:
+    s3_path = os.path.join(
+        os.getenv("METAFLOW_DATASTORE_SYSROOT_S3"),
+        flow_name,
+        kfp_run_id,
+        "_s3_sensor",
+    )
+    s3_path_parsed: ParseResult = urlparse(s3_path)
+    bucket: str = s3_path_parsed.netloc
+    key: str = s3_path_parsed.path.lstrip("/")
+    return bucket, key
+
+
+def read_elapsed_time_s3_path(flow_name: str, kfp_run_id: str) -> float:
+    bucket: str
+    key: str
+    bucket, key = construct_elapsed_time_s3_bucket_and_key(flow_name, kfp_run_id)
+    s3: botocore.client.BaseClient
+    s3_client_error: ClientError
+    s3, s3_client_error = get_s3_client()
```

Comment:
s3_client_error looks unused. I see it is simply an imported class, not a variable here:
https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/plugins/aws/aws_client.py#L11
Metaflow code use it like this:
https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/datastore/s3.py#L86

Can you change the name to reflect that it is an error class, or simply use `_` place holder to avoid this return value? Same for `write_elapsed_time_s3_path` and `wait_for_s3_path`?